bin/download-document-group.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+/**
+ * to run download-document-group applet from the project root folder type in your console:
+ * > node bin/download-document-group <client_id> <client_secret> <username> <password> <document_group_id> <path_to_save> <type> <with_history> <document_order>
+ * <client_id>, <client_secret>, <username>, <password>, <document_group_id>, <path_to_save> - are required params
+ * <type>, <with_history>, <document_order> - are optional params
+ */
+
+'use strict';
+
+const [
+  clientId,
+  clientSecret,
+  username,
+  password,
+  documentGroupId,
+  pathToSaveFile,
+  type,
+  withHistory,
```

Comment:
make `withHistory` as CLI command options. Example:
```
--with-history=after-each-document
or
--with-history=after-merged-pdf
or
--with-history=no
```
if option is omitted  use `no` as default